src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -76,16 +76,43 @@
         (rr/status ((response :outcome) outcomes->codes))
         (rr/content-type "text/plain"))))
 
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;;; Web app
-
 (defn try-to-parse
   [body]
   (try
     (cheshire/parse-stream (io/reader body) true)
     (catch Exception e
       (log/debug e))))
 
+(schema/defn handle-cert-clean
+  [{:keys [body]}
+   ca-settings :- ca/CaSettings]
+  (let [{:keys [certnames async]} (try-to-parse body)]
```

Comment:
I think we need some better error handling here. The exception from malformed JSON was swallowed and it printed a success message >.>